convert2rhel/backup.py

```diff
@@ -314,7 +317,19 @@ def restore(self):
             # IOError for py2 and OSError for py3
             loggerinst.warning("Error(%s): %s" % (err.errno, err.strerror))
             return
-        loggerinst.info("File %s restored." % self.filepath)
+
+        if rollback:
+            loggerinst.info("File %s restored." % self.filepath)
+        else:
+            loggerinst.debug("File %s restored." % self.filepath)
+
+    def remove(self):
+        """Remove restored file from original place, backup isn't removed"""
+        try:
+            os.remove(self.filepath)
+            loggerinst.debug("File %s removed." % self.filepath)
+        except (OSError, IOError, TypeError):
```

Comment:
I'm wondering why we need to have the `TypeError` here. Could you elaborate on why it is needed to catch this exception too? 

If it is something way off that you have experienced in your tests, then it might be good to add a comment explaining the reason to it.

Reply:
@r0x0d The `TypeError` exception would be raised, if the `filepath` would somehow contain `None`. Which I think couldn't happen, but if yes, I've choose that just saying `Couldn't remove restored file` is better than crashing on this.

Reply:
Right. Makes sense from this point of view, but since we construct the `RestorableFile` object from hard-coded strings in our code, then I don't think we need the `TypeError` catch here, as we will always pass an `filepath` to the constructor of the class.

Catching the `TypeError` in this function, for me, seems unnecessary, as it would be better if we catch that type of mistaken in the class constructor, for example: 

```python
class RestorableFile(object):
    def __init__(self, filepath):
        if not isinstance(filepath, str):
            raise TypeError("filepath must be an instance of str.")
            
        self.filepath = filepath
```

This way we could prevent working with an empty `filepath` in other methods inside the class.

But for now, I would say this is not strictly necessary, as we don't have any case in our code base that could pass an empty `filepath` to this class constructor.

Reply:
Ok, so I'll remove that.